ui_framework/src/components/header/_header_item.scss

```diff
@@ -0,0 +1,23 @@
+// Header items are the small icon links that pop up menus.
+
+@include component('kuiHeaderItem') {
+  position: relative;
+
+  // Add a pseudo-border to the left
+  @include kuiHeaderSeparation('left');
+
+  @include child('button') {
+    height: $kuiHeaderChildSize;
+    min-width: $kuiHeaderChildSize;
+    text-align: center;
+
+    &:hover {
+      background: $kuiColorLightestGray;
+    }
+
+    &:focus {
+      background: $kuiFocusBackgroundColor;
+    }
+  }
+}
```

Comment:
Continuing from the preceding comment, we can add a modifier to put the separator on the left side too, for the search button:

```scss
@include modifier('enclosed') {
  @include kuiHeaderSeparation('left');
}
```